packages/console/src/components/CreateConnectorForm/Footer/index.tsx

```diff
@@ -70,13 +88,19 @@ function Footer({
           <Trans
             components={{
               a: <ContactUsPhraseLink />,
-              planName: <PlanName name={planName} />,
+              planName: (
+                <PlanName
+                  name={conditional(isDevFeaturesEnabled && currentPlan.name) ?? planName}
+                />
```

Comment:
It seems `currentPlan.name` IS `planName`?

Reply:
Then you need to search and replace globally as I've seen more than one occurrances

Reply:
Yes, double checked the use cases of this component, thanks for reminding me.